CODING_STANDARDS.md

```diff
@@ -0,0 +1,76 @@
+# Coding standards & convention
+
+We have decided to create and follow some guidelines and best practices during development of Capybara theme. There are a few core goals that we stick to because we belive they improve our cooperation and readability of source code we've created.
+
+## Atomic Design rules
+
+[Atomic Design](https://atomicdesign.bradfrost.com/) is a methodology for creating user interfaces in structured and hierarchical manner. It is composed from 5 *elements* (or *stages*): atoms, molecules, organisms, templates and pages. Here in Capybara we have slightly adapted Atomic Design rules to our needs and we are using all of them except templates, because we've not found any use case for them. Each of those elements play a key role in designing our interface. In the `/components` folder we created a dedicated sub-folder for atoms, molecules and organisms. Each `.vue` file inside them has a name prefixed by `a-` for atoms, `m-` for molecules and `o-` for organisms. Similarly, the name for every our component follow the same naming convention: it is prefixed by `A` for atoms, `M` for molecules and `O` for organisms. This approach helps us categorize and figure out very fast what kind of component we see in the markup. Pages on the other hand are located in dedicated `/pages` folder. Because they are located in a separate location and they are not used within templates like ordinary components, they do not need to be prefixed.
```

Comment:
Typo: is "composed from" and should be "composed of"

Reply:
Fixed.